db/migrate/20161118210435_add_question_number.rb

```diff
@@ -0,0 +1,6 @@
+class AddQuestionNumber < ActiveRecord::Migration
+  def change
+    add_column :questions, :number, :integer, unique: true
```

Comment:
Why does `question_order` have `survey_id` again? 

What if `survey_id` was on `question` instead and would it be able to provide the same functionality?